Makefile

```diff
@@ -0,0 +1,10 @@
+all: grpc
+
+grpc:
+	python -m grpc_tools.protoc --proto_path=`pwd`/quarkchain/proto --python_out=`pwd`/quarkchain/generated --grpc_python_out=`pwd`/quarkchain/generated  grpc.proto
```

Comment:
no extra space between commands

Reply:
also, seems no need to add `pwd` as the default working directory is current folder?